### ISSUE TITLE
Mention the need for gmake

### ISSUE THREAD

**Issue by dkorpel** [CyberShadow/Digger#91]
I used Digger for the first time today on my Debian box, and encountered the issue that std.process could not find the executable 'gmake'. After some Googling I found out that it's simply supposed to be a symlink to GNU make, so I fixed it by doing this:

```
cd /usr/bin/ && sudo ln -s make gmake
```

This need for 'gmake' is not mentioned in the readme, maybe it's a good idea to do so?

**Comment by CyberShadow** [comment]
It was an incorrect assumption introduced in https://github.com/cybershadow/ae/commit/8bad002a2e70f4aa1af6413a5cb4b28bc05b6770 but then shortly fixed in https://github.com/cybershadow/ae/commit/e5dddeb076cbe21b62e0182a3a4b625da25f7f14. Digger v3.0.3 includes 0700c6d1370e7e00a719190af693e015b58d3060, which updates ae to the version with the fix (it never referenced the version of ae with this bug).

How are you building Digger? And, I assume this is on Debian GNU/Linux and not Debian GNU/kFreeBSD or such?

**Comment by dkorpel** [comment]
```
dub fetch digger
dub run digger
Building package digger in /home/dennis/.dub/packages/digger-3.0.3/digger/
```

I'm running Debian GNU/Linux version 10 (buster).

**Comment by CyberShadow** [comment]
Found the problem, there was a bug in the tagging script causing the tags to not match the submodule.

Should be fixed in Digger 3.0.4.
